### PR TITLE
blog: grants and language blogpost

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by krzysztof** [comment]
<a href="https://i.imgur.com/P0Zhg79.png">
  
</a>

File: content/blog/grants-and-language-update/contents.lr [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+2. Language field
+3. ORCID for authors
+
+## Extended grant support (powered by OpenAIRE)
+
+We are expanding our grants database with over 620,000 grants from 8 new funders such as National Science Foundation (US) and Wellcome Trust (UK) - all thanks to [OpenAIRE](https://openaire.eu/)'s ever growing grants database.
+
+![Zenodo OpenAIRE grants database](grants.png)
+
+So far we have been supporting only grants from the European Commission (FP7 and Horizon 2020). Today our dataset contains grants from the following funders:
```
Comment (TimSmithCH) [inline content/blog/grants-and-language-update/contents.lr]: So far we have **only** been supporting ~~only~~ grants
Today our dataset **has grown and** contains grants

File: content/blog/grants-and-language-update/contents.lr [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* European Commission (EU) - 39409 grants
+* Foundation for Science and Technology (Portugal) - 37277 grants
+* National Health and Medical Research Council (Australia) - 24354 grants
+* Netherlands Organisation for Scientific Research (The Netherlands) - 24180 grants
+* Australian Research Council (Australia) - 23011 grants
+* Wellcome Trust (United Kingdom) - 12196 grants
+* Ministry of Science and Education (Croatia) - 2120 grants
+* Ministry of Education, Science and Technological Development (Serbia) - 777 grants
+
+which means that our grants database grew from close **40,000** to over **660,000** grants! We wouldn't be able to do that if it wasn't for the OpenAIRE team and their hard work in collecting, maintaining and distributing the grants database for the benefit of Open Science!
```
Comment (TimSmithCH) [inline content/blog/grants-and-language-update/contents.lr]: our grants database grew from ~~close~~ **nearly**
We wouldn't **have been** able to do

File: content/blog/grants-and-language-update/contents.lr [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Language field
+
+Today we are also adding a new language field to our metadata, which allows you to record the primary language of an upload. You can select the language in the upload form simply by starting to type the English name, a 2-letter or a 3-letter ISO 639 code:
+
+![Language for Zenodo](language.png)
+
+The new language field supports all languages defined in ISO 639-3, which in total defines 436 individual and macro-languages. For full reference of language codes see [Library of Congress ISO 639 Language List](https://www.loc.gov/standards/iso639-2/php/code_list.php).
```
Comment (TimSmithCH) [inline content/blog/grants-and-language-update/contents.lr]: For **the** full reference of

File: content/blog/grants-and-language-update/contents.lr [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+#### What if my record contains more than one language?
+
+The field is used to specify the primary language of the resource, hence if, e.g., a thesis is written in Danish and has an English abstract, then the primary language is Danish. Similarly, the primary language of a paper written in English, which is on the topic of Greek linguistics, thus containing a lot of text in Greek, is English.
+
+There are always cases where it's not possible to clearly determine the primary language, for example for a dataset containing the mapping between common Polish and French phrases. In those special cases you can always use ISO 639-3 code [`mul` (Multiple Languages)](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?code_ID=306).
```
Comment (TimSmithCH) [inline content/blog/grants-and-language-update/contents.lr]: ~~it's~~ **it is**

**Comment by krzysztof** [comment]
@TimSmithCH Thank you! I've applied the changes.

File: content/blog/grants-and-language-update/contents.lr [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+body:
+Today, we are introducing three additions to Zenodo:
+
+1. Extended grant support
+2. Language field
+3. ORCID for authors
+
+## Extended grant support (powered by OpenAIRE)
+
+We are expanding our grants database with over 620,000 grants from 8 new funders such as National Science Foundation (US) and Wellcome Trust (UK) - all thanks to [OpenAIRE](https://openaire.eu/)'s ever growing grants database.
```
Comment (slint) [inline content/blog/grants-and-language-update/contents.lr]: Shouldn't it be "**the** National Science Foundation (US) and **the** Wellcome Trust (UK)"?

File: content/blog/grants-and-language-update/contents.lr [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+which means that our grants database grew from nearly **40,000** to over **660,000** grants! We wouldn't have been able to do that if it wasn't for the OpenAIRE team and their hard work in collecting, maintaining and distributing the grants database for the benefit of Open Science!
+
+## Language field
+
+Today we are also adding a new language field to our metadata, which allows you to record the primary language of an upload. You can select the language in the upload form simply by starting to type the English name, a 2-letter or a 3-letter ISO 639 code:
+
+![Language for Zenodo](language.png)
+
+The new language field supports all languages defined in ISO 639-3, which in total defines 436 individual and macro-languages. For the full reference of language codes see [Library of Congress ISO 639 Language List](https://www.loc.gov/standards/iso639-2/php/code_list.php).
```
Comment (slint) [inline content/blog/grants-and-language-update/contents.lr]: Similar: "For full reference of language codes see **the** [Library of Congress ISO 639 Language List]"

**Pull request merged**